### PR TITLE
Separating MultiArch Libvirt Jobs

### PR DESCRIPTION
**Pull request by deepsm007**
Branching out the remote libvirt jobs including the image ecosystem and jenkins

### CODE REVIEW

**Comment by bparees** [comment]
everything else under the jobs dir is nested as
jobs/orgname/reponame

this introduces a pattern of:
jobs/grouping_name

@openshift/openshift-team-developer-productivity-test-platform is that problematic for any of the tooling? or general violation of expected patterns?

(it's also, i think, the only set of jobs that are defined which are not tied to a github org+repo)

**Comment by jaypoulz** [comment]
Besides the rebase, I worry that the rehearsal jobs didn't get auto-generated. I wonder if it's related to the inconsistency mentioned by Ben in the comment above.
I would also like this PR to to be reviewed by Steve Kuznetsov or someone from DPTP before this merges.

**Review by jaypoulz — APPROVED** [review]
I'm not surprised here that a bunch of the tests failed when they flooded the available environments. :)

Everything appears to be operating normal, but we should continue to monitor once this lands.

**Comment by jaypoulz** [comment]
@deepsm007 The last step for this PR will be to have it reviewed by a member of the DPTP team. (See the owners file).

**Comment by petr-muller** [comment]
What's the reason for doing this? Having `OWNERS`?
I'm a bit worried that these will get off the radar when we are doing e.g. cuts and other release jobs maintenance...

**Comment by jaypoulz** [comment]
@petr-muller yes - this was about giving multi-arch the ability to ack our own job configs, considering how often the skip list changes.

The PR spawned from this discussion on slack, for context.
https://coreos.slack.com/archives/CBN38N3MW/p1610558550339000?thread_ts=1610489122.310900&cid=CBN38N3MW

File: ci-operator/jobs/openshift/multiarch/OWNERS
Comment (petr-muller) [inline ci-operator/jobs/openshift/multiarch/OWNERS:7]: I don't think DPTP wants to be an explicit approver or reviewer here. We do not own the jobs.

**Comment by petr-muller** [comment]
@jaypoulz Yeah, that makes sense, but moving them out of "central" location also means that they get off everyone's radar. Do you commit to their maintenance - e.g. creating new jobs for new releases, such as 4.8 that happened recently?

@stevekuznetsov WDYT?

**Comment by jaypoulz** [comment]
@petr-muller 4.8 jobs aren't created automatically already. With the exception of the image-ecosystem tests.

We manually create the job configurations for each new release, because the skip list changes per release.

**Comment by petr-muller** [comment]
Perfect.

/approve

**Pull request merged**
